erpnext/patches/v7_0/create_warehouse_nestedset.py

```diff
@@ -0,0 +1,10 @@
+import frappe
+from frappe import _
+
+def execute():
+	for warehouse in frappe.db.sql_list("""select name from tabWarehouse
+		order by company asc, name asc"""):
+		warehouse = frappe.get_doc("Warehouse", warehouse)
+		warehouse.is_group = "No"
+		warehouse.parent_warehouse = ""
```

Comment:
Create new warehouse group called "All Warehouses" and put all existing warehouses under it.